src/docs/product/integrations/github/index.mdx

```diff
@@ -7,18 +7,16 @@ redirect_from:
   - /workflow/global-integrations/integrations/github/
 ---
 
-You can use the data from your GitHub commits to help find and fix bugs faster.
+Track and resolve bugs faster by using data from your GitHub commits, and streamline your triaging process by creating a GitHub issue directly from Sentry.
 
-## Configure GitHub
+## Install
 
 <Alert title="Note" level="info">
   Sentry owner or manager permissions, and GitHub owner permissions are required
   to install this integration.
 </Alert>
 
-### GitHub (github.com)
-
-1. In Sentry, navigate to Organization Settings > **Integrations**.
+1. In Sentry, navigate to **Organization Settings > Integrations**.
 2. If you have the legacy GitHub integration installed, you’ll see a button next to GitHub that says **Upgrade**. If you do not have the legacy GitHub integration installed, you'll see a button that says **Install**.
```

Comment:
see previous suggestion to integrate the button names directly into the language of the sentence.